### PR TITLE
Adjust Comment Text Wrapping To Avoid Overflow

### PR DESCRIPTION
**Pull request by kazma-ku**

### CODE REVIEW

**Comment by kazma-ku** [comment]
I accidentally pushed files that are not directly related to the issue. There doesn't seem to be any conflict, but let me know if I need to revert it. Other than that, it seems to work fine.

File: server/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   },
   "homepage": "https://github.com/MyLivingCity/my-living-city#readme",
   "dependencies": {
-    "@prisma/client": "^2.30.3",
+    "@prisma/client": "^4.13.0",
```
Comment (oscarlaaaa) [inline server/package.json:27]: Oh are we updating the Prisma and Nodemon versions in this PR?

File: frontend/src/components/partials/SingleIdeaContent/CommentsSection.tsx [hunk 2]
```diff
@@ -101,9 +104,13 @@ if(ideaComments){
         ) : (
           ideaComments &&
           ideaComments.map((comment) => (
+            <div className="textlimit & textlimit:hover">
```
Comment (oscarlaaaa) [inline frontend/src/components/partials/SingleIdeaContent/CommentsSection.tsx]: What's the point of the :hover portion of the classname? Do comments have some sort of hover functionality?

Also, just curious: what does the & do here? I've never seen this before and I can't find anything about it online

**Review by oscarlaaaa — APPROVED** [review]
Overall things look good; I just have a few questions regarding some things but overall it works fine on my end.
images

File: server/package.json (hunk 1 above)
Comment (kazma-ku) [reply server/package.json:27]: I think that was an automatic update when installing Prisma on my end. I'll check if it works with the previous version.

File: frontend/src/components/partials/SingleIdeaContent/CommentsSection.tsx (hunk 2 above)
Comment (kazma-ku) [reply frontend/src/components/partials/SingleIdeaContent/CommentsSection.tsx]: That was something I forgot to delete and I've updated it. Thanks for catching it!!

**Pull request merged**